target/linux/ramips/dts/mt7621_wevo_11acnas.dts

```diff
@@ -6,10 +6,16 @@
 	compatible = "wevo,11acnas", "wevo,w2914ns-v2", "mediatek,mt7621-soc";
 	model = "WeVO 11AC NAS Router";
 
+	aliases {
+		led-boot = &led_usb;
+		led-failsafe = &led_usb;
+		led-upgrade = &led_usb;
```

Comment:
The LED used here should not be mistakable. Using USB, WiFi, LAN LEDs etc. is not a good idea.

So, either remove this entirely or at least make this a separate commit, so the rest is not delayed by this discussion.

Reply:
Hmm... There's no device using USB LED for status indication so you may be right, but I still think it will be very helpful to have a status LED, at least during boot. I'll separate it as another commit then.